### DISCUSSION TITLE
Demand for the inclusion of the tower base bending moment signal in the AvrSWAP array for controller design

### DISCUSSION THREAD

**Discussion by fengguoFUAS** [IEAWindTask52/LidarAssistedControl#16] [Ideas]
Hi all, 

I am working on the 30 s Sprint for the IEA LAC Summer game 2024. I would like to have the tower base bending moment signal in the AvrSWAP array for my controller design.

I am curious how many of you also have this demand? It would be good to know you thoughts about this point.

BR
Feng Guo
PostDoc
Shanghai Jiao Tong University

**Comment by DavidSchlipf** [comment]
Hi Feng, 

Thanks a lot for participating and your question! Looking forward to your contribution!

For including the tower base bending moment signal, you would need to recompile the OpenFAST, right? 
We haven't really thought about that when designing the summer games: https://zenodo.org/records/10931629

For the 30 s sprint, I am not sure: On the one hand the intention was to trigger new control concepts, and maybe it is better, if one can focus on the control problem alone without state estimation. On the other hand, using only the existing OpenFAST executable would be easier. 

For the DLC 1.2 Marathon, I do have a stronger opinion: I would find it better to only focus on the usual turbine signals used for wind turbine control (generator speed, pitch angle, tower acceleration), since the intention here is to benchmark concepts.

What do the others think?

**Reply by fengguoFUAS** [reply]
Hi David,

I just considered this issue again.

For the 30 s sprint, I think it is easier to use perfect state for now, since implementing state estimation can be another topic.  We can also integrate the tower acc and remove the perfectly known initial value to get the almost perfect tower top speed and deflection. 

For the DLC 1.2, I agree that we should make the results more close to industrial relizations and avoid using perfect states.

BR
Feng Guo
PostDoc
Shanghai Jiao Tong University
